xcode/Subconscious/Shared/Components/Common/Profile/UserProfileView.swift

```diff
@@ -40,6 +40,8 @@ struct RecentTabView: View {
                     ),
                     action: { address, _ in onNavigateToNote(address) }
                 )
+                
+                Divider()
```

Comment:
I moved the dividers out of the story components so I could have more flexibility in rendering them based on context.